lib/Devel/KYTProf/Profiler/Furl/HTTP.pm

```diff
@@ -8,13 +8,32 @@ sub apply {
         'Furl::HTTP',
         'request',
         sub {
-            my($orig, $self, %args) = @_;
+            my ($orig, $self, %args) = @_;
+
+            my $method = $args{method} || 'GET';
```

Comment:
[memo]
Even inside Furl, when method is empty uses GET method 
https://github.com/tokuhirom/Furl/blob/537c1e6a65a5e32f6ac3d25f235831f9319308f1/lib/Furl/HTTP.pm#L324